static/css/styles.scss

```diff
@@ -348,13 +344,12 @@ a.subheading-link {
   padding-right: 15px;
   vertical-align: middle;
   background-image: none;
-  font-size: 12pt;
```

Comment:
Semi-related, but the CSS would be 17% easier to read if we sorted all the properties (plus easier to see these duplicate-but-not-quite-duplicate) rules. But I avoided sorting everything because it would have been a super-messy PR with virtually every line of CSS moving around.

I can do that in a future PR where I'm not also trying to change values.